modules/core/arrow-core/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -90,7 +85,7 @@ interface EitherApply<L> : Apply<EitherPartialOf<L>>, EitherFunctor<L> {
   override fun <A, B> EitherOf<L, A>.map(f: (A) -> B): Either<L, B> = fix().map(f)
 
   override fun <A, B> EitherOf<L, A>.ap(ff: EitherOf<L, (A) -> B>): Either<L, B> =
-    fix().eitherAp(ff)
+    fix().ap(ff)
```

Comment:
This is going to cause a stackOverflow because you'd think it calls its member function but no. It calls itself instead.